I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 186, in script call
    call _start_store
  File "renpy/common/00start.rpy", line 139, in script call
    label _start_store:
  File "renpy/common/00start.rpy", line 94, in script
    python hide:
  File "renpy/common/00start.rpy", line 94, in <module>
    python hide:
  File "renpy/common/00start.rpy", line 97, in _execute_python_hide
    i()
  File "renpy/common/00console.rpy", line 600, in watch_after_load
    if config.developer and traced_expressions:
NameError: global name 'traced_expressions' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/00start.rpy", line 186, in script call
    call _start_store
  File "renpy/common/00start.rpy", line 139, in script call
    label _start_store:
  File "renpy/common/00start.rpy", line 94, in script
    python hide:
  File "/Users/James/Desktop/renpy-6.99.14.3-sdk/renpy/ast.py", line 862, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Users/James/Desktop/renpy-6.99.14.3-sdk/renpy/python.py", line 1888, in py_exec_bytecode
    exec bytecode in globals, locals
  File "renpy/common/00start.rpy", line 94, in <module>
    python hide:
  File "renpy/common/00start.rpy", line 97, in _execute_python_hide
    i()
  File "renpy/common/00console.rpy", line 600, in watch_after_load
    if config.developer and traced_expressions:
NameError: global name 'traced_expressions' is not defined

Darwin-15.6.0-x86_64-i386-64bit
Ren'Py 6.99.14.3.3347
RenPy 1.0
Thu May 24 18:18:37 2018
